manifest/services.py

```diff
@@ -32,11 +32,18 @@ def get_manifest_service(args=None, configuration=None):
             logger.error(msg)
             raise ManifestServiceException(msg)
         manifest_config = configuration.config.manifest
-        manifest_config.update({"output_dir": configuration.output_dir, "gcp_bucket": args.gcp_bucket})
-        logger.debug("Initializing Manifest service, configuration: {}".format(json.dumps(manifest_config)))
+        manifest_config.update(
```

Comment:
I think it would be interesting to have a conversation about formatting, in case we could find the use of a linter a useful thing to have